blog/content/post/2020-10-19-security-part-2.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Application Security as a Every Developer's Responsibility
+abstract: "The issue of web application security is a topic that is becoming more and more popular. Currently, not only large organizations and corporations, but also smaller and smaller businesses are forced by progress to transfer some of their activities to the Internet."
+cover: /covers/image-file.png
+author: piotr.nowoslawski
+layout: post
+published: true
+date: 2020-10-19
+archives: "2020"
+
+tags:
+  - security
+  - application security
+categories:
+  - Development
+  - App Dev
+  - Security
+---
+
+The issue of web application security is a topic that is becoming more and more popular. Currently, not only large organizations and corporations, but also smaller and smaller businesses are forced by progress to transfer some of their activities to the Internet. This process has accelerated even more because of the current popularity of all electronic devices and changes in the way people communicate remotely.
+
+While large business entities have been operating in the realities of application security for a long time, smaller entities have very little or no knowledge of how to adapt to these new, demanding conditions without exposing themselves to business risk. Unfortunately, the risk of operating in digital markets only increases. Due to increasingly stringent regulations, the loss or disclosure of data has very serious consequences. These can be financial penalties, loss of reputation or customer trust, which ultimately translates into a bad financial result, and in the worst case it will lead to the collapse of the company.
```

Comment:
```suggestion
While large business entities have been operating in the realities of application security for a long time, smaller entities have very little or no knowledge of adapting to these new, demanding conditions without exposing themselves to business risk. Unfortunately, the risk of operating in digital markets only increases. Due to increasingly stringent regulations, the loss or disclosure of data has severe consequences. These can be financial penalties, loss of reputation, or customer trust, finally turning into a bad financial result. In the worst case, it may lead to the collapse of the company.
```